wiki/Tournaments/112/Summer_2019/en.md

```diff
@@ -297,7 +295,7 @@ After Group matches, those who get through to Quarterfinals are obliged to form
 - Tiebreaker
   - **[Halozy – Paranoid Lost (Hakurei Yoru) \[Soda’s Lunatic\]](https://osu.ppy.sh/beatmapsets/440925#osu/964622)**
 
-### Qualifiers Stage
+### Qualifiers stage
```

Comment:
Super nitpicky, but you refer to it as just "Qualifiers" above, instead of "Qualifiers stage"
https://github.com/ppy/osu-wiki/pull/5861/files#diff-b0dd353174837641fb7324cc878b6c0a649fdb98cb7af1ea15681719ef0aa294R19